modules/ROOT/pages/appendices/troubleshooting.adoc

```diff
@@ -82,35 +83,7 @@ The ownCloud iOS app sends the `X-REQUEST-ID` header with every request. You'll
 `X-REQUEST-ID` to the logs. Here you can find more information at
 {owncloud-log-tracing-url}[Request Tracing]
 
-Some helpful files include the following:
```

Comment:
did this get accidentally deleted?

Reply:
Well, it's been removed from 12.0 and master earlier. @michaelstingl please check if this should stay for 11.11.

Reply:
Doesn't make sense to elaborate on web server internals in the iOS docs. Can be removed 👍